src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -23,6 +23,8 @@ import AddTaskButton from './AddTaskButton';
 import SidebarUtils from './SidebarUtils';
 import { DEFAULT_COLLAB_DEBOUNCE, ORIGIN_ACTIVITY_SIDEBAR, SIDEBAR_VIEW_ACTIVITY } from '../../constants';
 import './ActivitySidebar.scss';
+import type { TaskType, TaskNew } from './activity-feed/task-new/flowTypes';
```

Comment:
It might be nice to import these into a sidebar-specific types module, then re-export them as a single object. That way, we wouldn't need to import types from all over the place directly in our components. Thoughts?